app/assets/javascripts/header.js

```diff
@@ -0,0 +1,14 @@
+document.addEventListener("DOMContentLoaded", function (event) {
+  let navbar=$(".navbar-default");
```

Comment:
Please be more careful with spacing here. 

```javascript
  let navbar = $(".navbar-default");
```
See the spaces. Do the same for the if below, and the else. 

You should consider installing js hint or js lint addons to your editor.